contract-message-predicate/Forc.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 authors = ["Fuel Labs <contact@fuel.sh>"]
-entry = "main.sw"
+entry = "contract_message_predicate.sw"
```

Comment:
Unsure about the name `contract_message_predicate`. Is it a contract; a predicate? Is this "message" some part of a contract or the predicate itself?

Reply:
It's a shorthand of "The predicate for sending a message to a contract (aka. a contract message)"